src/styles/themeManager.js

```diff
@@ -0,0 +1,257 @@
+      actionColor: palette.accent1Color,
+    },
+    table: {
+      backgroundColor: palette.canvasColor,
+    },
+    tableHeader: {
+      borderColor: palette.borderColor,
+    },
+    tableHeaderColumn: {
+      textColor: palette.accent3Color,
+      height: 56,
+      spacing: 24,
+    },
+    tableFooter: {
+      borderColor: palette.borderColor,
+      textColor: palette.accent3Color,
+    },
+    tableRow: {
+      hoverColor: palette.accent2Color,
+      stripeColor: ColorManipulator.lighten(palette.primary1Color, 0.55),
+      selectedColor: palette.borderColor,
+      textColor: palette.textColor,
+      borderColor: palette.borderColor,
+    },
+    tableRowColumn: {
+      height: 48,
+      spacing: 24,
+    },
+    timePicker: {
+      color: palette.alternateTextColor,
+      textColor: palette.accent3Color,
+      accentColor: palette.primary1Color,
+      clockColor: palette.textColor,
+      clockCircleColor: palette.clockCircleColor,
+      headerColor: palette.pickerHeaderColor || palette.primary1Color,
+      selectColor: palette.primary2Color,
+      selectTextColor: palette.alternateTextColor,
+    },
+    toggle: {
+      thumbOnColor: palette.primary1Color,
+      thumbOffColor: palette.accent2Color,
+      thumbDisabledColor: palette.borderColor,
+      thumbRequiredColor: palette.primary1Color,
+      trackOnColor: ColorManipulator.fade(palette.primary1Color, 0.5),
+      trackOffColor: palette.primary3Color,
+      trackDisabledColor: palette.primary3Color,
+      labelColor: palette.textColor,
+      labelDisabledColor: palette.disabledColor,
+      trackRequiredColor: ColorManipulator.fade(palette.primary1Color, 0.5),
+    },
+    toolbar: {
+      backgroundColor: ColorManipulator.darken(palette.accent2Color, 0.05),
+      height: 56,
+      titleFontSize: 20,
+      iconColor: 'rgba(0, 0, 0, .40)',
+      separatorColor: 'rgba(0, 0, 0, .175)',
+      menuHoverColor: 'rgba(0, 0, 0, .10)',
+    },
+    tabs: {
+      backgroundColor: palette.primary1Color,
+      textColor: ColorManipulator.fade(palette.alternateTextColor, 0.6),
+      selectedTextColor: palette.alternateTextColor,
+    },
+    textField: {
+      textColor: palette.textColor,
+      hintColor: palette.disabledColor,
+      floatingLabelColor: palette.textColor,
+      disabledTextColor: palette.disabledColor,
+      errorColor: Colors.red500,
+      focusColor: palette.primary1Color,
+      backgroundColor: 'transparent',
+      borderColor: palette.borderColor,
+    },
+  };
+}
+
+//functions to modify properties of base theme, namely spacing, palette
+//and font family. These functions use the React update immutability helper
+//to create a new object for the base theme, and return a new MUI theme object
+
+/**
+ * Modifies the spacing of the base theme. This function recomputes
+ * the MUI theme and returns it based on the base theme.
+ */
+export function modifyBaseThemeSpacing(muiTheme, spacing) {
```

Comment:
Shouldn't we make those methods deprecated since we merge with the `baseTheme` with the `lightBaseTheme`?